tests/functional/test_lint_imports.py

```diff
@@ -59,6 +59,9 @@
         # Namespace packages
         (namespace_packages_directory, "keptcontract.ini", cli.EXIT_STATUS_SUCCESS),
         (namespace_packages_directory, "brokencontract.ini", cli.EXIT_STATUS_ERROR),
+        # Type checking imports
+        (testpackage_directory, ".typecheckkeptcontract.toml", cli.EXIT_STATUS_SUCCESS),
```

Comment:
Could we do these in ini format rather than toml format, as it's more consistent with the other functional tests?